### PR TITLE
Fix/1611/wrong config

### PR DESCRIPTION
**Pull request by tobiasqueck**
Issue: the ADP template created a wrong configuration of the `fiori-tools-proxy`, this PR is fixing it as well as a few linting issues.

### CODE REVIEW

File: packages/adp-tooling/test/unit/writer/__snapshots__/index.test.ts.snap [hunk 1]
```diff
@@ -80,18 +80,18 @@ server:
             url: http://sap.example
           ignoreCertErrors: false
     - name: fiori-tools-proxy
-      afterMiddleware: fiori-tools-preview
+      afterMiddleware: compression
       configuration:
-        ignoreCertErrors: false
+        ignoreCertError: false # If set to true, certificate errors will be ignored. E.g. self-signed certificates will be accepted
```
Comment (nikmace) [inline packages/adp-tooling/test/unit/writer/__snapshots__/index.test.ts.snap:85]: When did this comment get added: `# If set to true, certificate errors will be ignored. E.g. self-signed certificates will be accepted`? I think there were no comments in the ui5.yaml previously

**Review by nikmace — CHANGES_REQUESTED** [review]

File: packages/adp-tooling/test/unit/writer/__snapshots__/index.test.ts.snap (hunk 1 above)
Comment (tobiasqueck) [reply packages/adp-tooling/test/unit/writer/__snapshots__/index.test.ts.snap:85]: That comment was always generated for a Fiori app project but not for adp. By reusing the existing app code, it is now added here as well.

File: packages/adp-tooling/src/writer/options.ts [hunk 2]
```diff
@@ -98,36 +103,27 @@ function getFioriToolsMiddlwares(config: AdpWriterConfig): CustomMiddleware<unkn
...
+    ]);
+    ui5Config.addFioriToolsProxydMiddleware({
+        ui5: {
+            url: config?.ui5?.frameworkUrl,
+            version: config?.ui5?.minVersion ?? '' //default to latest if version is not set
+        },
+        backend: [backendConfig as FioriToolsProxyConfigBackend]
+    });
+
+    return;
```
[the remark this answers is not in the document]
Comment (tobiasqueck) [reply packages/adp-tooling/src/writer/options.ts]: That was a copy and paste issue. Fixed with 4824b76a523d81c51b798748155212b9e6a624d1

**Review by nikmace — APPROVED** [review]
Code looks good
Coverage is good
Did not test manually

**Review by Klaus-Keller — APPROVED** [review]
- code fix looks good
- changeset exists
- test coverage is good
- did not run a local test, just visual review

Approved from my side

**Pull request merged**